NEWS.rst

```diff
@@ -5,29 +5,131 @@ NEWS
 HEAD
 ====
 
+Features
+--------
+
+- Core
+  - New simplified policy language (670_)
+  - Option to choose between a global and per endpoint connection tracking table (659_)
+  - Parallel endpoint BPF program & policy builds (424_, 587_)
+  - fluentd logging integration (758_)
+  - IPv6 proxy redirection support (818_)
+  - Transparnt ingress proxy redirection (773_)
+  - Consider all labels for identity except dynamic k8s state labels (849_)
+  - Reduced size of cilium binary from 27M to 17M (554_)
+  - Add filtering support to ``cilium monitor`` (673_)
+  - Allow rule now supports matching multiple labels (638_)
+  - Separate runtime state and template directory for security reasons (537_)
+  - Ability to specify L4 destination port in policy trace (650_)
+  - Improved log readability (499_)
+  - Optimized connection tracking map updates per packet (829_)
+  - New --kvstore and --kvstore-opt flag (767_)
```

Comment:
`  - New --kvstore and --kvstore-opt flag (767_) (Replaces ``--consul``, ``--etcd`` and ``--local`` flags)